### ISSUE TITLE
ore distribution unbalanced?

### ISSUE THREAD

**Issue by dskw** [WesCook/Homestead#49]
Hi,

I'm currently automating my mining using a high level steve's cart design but I have a really hard time finding any copper in the world.
The gold veins are drowning me in gold, plus nearly every mob dropping discs that convert to two nuggets and sometimes dropping full ingots.

Iron is rarer than in vanilla, which IMHO is totally fine. After some time I managed to get usable results with automation, but the only copper I found so far was 4 ores in an overworld cave entrance and some ingots in a village, nothing more.
And that by the way after nearly 2 weeks of playing and having completed >80% of the questbook. It was barely enough to build two dynamos and a metal press. Had so go for MV cables right away since I did not have any copper left but electrum easily available.

So in short:
* drowning in gold
* copper nearly not available
* all other ores are ok with silver being more rare and aluminium and lead between iron and silver

What height level is set for the ore gen for copper and is it maybe too scarce?

PS: I also noticed that the ore-extractor module from Steve's Carts is not mining the WTF ores in the walls, maybe not compatible?

**Comment by WesCook** [comment]
Thanks for the information.  I was thinking of doing another pass on ore distribution balancing.  I'll look into this before the next update.

Copper I believe is set to generate higher in the world, however it's not tied to a specific Y level.  It's actually a percentage of the height of that area.  So mountainous areas will scale the values higher than a desert might.

**Comment by dskw** [comment]
Hey, thanks for looking into it :)
I managed to find a decent amount of copper by now, but you are right, it is set to spawn very high and in my plains biome it spawns nearly one level below the top dirt layer, which forces me to dig up all the area with my cart.
Same with forest and other biomes. The occasional mountain is too small for proper mining in my case so I did not try it yet.
But regardless of the level the distribution is rather low considering that it is the entry level resource for the IE energy distribution.

**Comment by WesCook** [comment]
I've made the suggested balance changes in Homestead v1.7.  Gold has been toned down, and iron and copper made more common.  Thanks again for the feedback!
